src/routes/profile.ts

```diff
@@ -7,6 +8,9 @@ import {
   deleteProfile,
 } from "../controllers/profile";
 
+const upload = multer({ dest: 'uploads/' });
```

Comment:
what is this destination? are you planning on storing pics inside the project?

Reply:
we won't plan to store our photos here. this folder serves as an intermediate link between saving the download from the front and transferring it to the claudinary

Reply:
@Enigma2017  so after they are uploaded to Cloudinary, the folder will be emptied?

Reply:
yes, it will be emptied